src/js/view/controls/components/tooltipicon.js

```diff
@@ -61,6 +61,7 @@ export default class Tooltip {
             this.trigger('open-' + this.componentType, evt, { isOpen: true });
             this.isOpen = true;
             toggleClass(this.el, this.openClass, this.isOpen);
+            toggleClass(this.tooltip, this.openClass, this.isOpen);
```

Comment:
why does the tooltip need the open class now ? in the css we only check for `jw-horizontal-volume-container.jw-open`